tests/kernel/timer/timer_api/src/main.c

```diff
@@ -92,7 +92,26 @@ static void status_stop(struct k_timer *timer)
 	TIMER_ASSERT(k_timer_remaining_get(timer) == 0, timer);
 }
 
-/* test cases */
+/**
+ * @brief Tests for the Timer kernel object
+ * @defgroup kernel_timer_tests Timer
+ * @ingroup all_tests
+ * @{
```

Comment:
instead of including everything, which will add non-tests, close the curly bracket here and add each test individually to the group with

`@ingroup kernel_timer_tests`